lib/views/teach_word_view.dart

```diff
@@ -235,11 +273,12 @@ class _TeachWordViewState extends State<TeachWordView>
                                       color: Color.fromRGBO(245, 245, 220, 100),
                                       isSelected: controller.showOutline,
                                       icon: const Icon(
-                                        Icons.remove_red_eye_outlined),
+                                          Icons.remove_red_eye_outlined),
                                       selectedIcon:
-                                        const Icon(Icons.remove_red_eye),
+                                          const Icon(Icons.remove_red_eye),
                                       onPressed: () {
-                                        controller.setShowOutline(!controller.showOutline);
+                                        controller.setShowOutline(
+                                            !controller.showOutline);
```

Comment:
Align to the 2-space indent style with previous codes. Remove additional indents from 4 to 2.